lss-manage-side-menu.html

```diff
@@ -107,85 +124,108 @@
 			<div class="side-nav-section-end"></div>
 
 			<div class="side-nav-section" category="1">Auditing</div>
-			<a name="log-parser" parent-category="1" display-role="View All Logs" href$="https://[[devPrefix]]manage.leavitt.com/log-parser/apps">
+			<a name="log-parser" parent-category="1" display-role="View All Logs"
+				href$="https://[[devPrefix]]manage.leavitt.com/log-parser/apps">
 				<iron-icon icon="nav:receipt"></iron-icon>Log Parser
 			</a>
 			<div class="side-nav-section-end" category="1"></div>
 
 			<div class="side-nav-section" category="2">Employee Management</div>
-			<a name="employee-manager" parent-category="2" display-role="Employee Manager Global Admin" href="https://[[devPrefix]]manage.leavitt.com/employee-manager/view/all/">
+			<a name="employee-manager" parent-category="2" display-role="Employee Manager Global Admin"
+				href="https://[[devPrefix]]manage.leavitt.com/employee-manager/view/all/">
 				<iron-icon icon="nav:account-settings"></iron-icon>Employee Manager
 			</a>
-			<a name="hire-form" parent-category="2" display-role="Hire Employee" href$="https://[[devPrefix]]manage.leavitt.com/hire-form/step1">
+			<a name="hire-form" parent-category="2" display-role="Hire Employee"
+				href$="https://[[devPrefix]]manage.leavitt.com/hire-form/step1">
 				<iron-icon icon="nav:add-circle"></iron-icon>Hire
 			</a>
-			<a name="job-role-types-manager" parent-category="2" display-role="Job Role Manager Global Admin" href$="https://[[devPrefix]]manage.leavitt.com/job-role-types-manager">
+			<a name="job-role-types-manager" parent-category="2" display-role="Job Role Manager Global Admin"
+				href$="https://[[devPrefix]]manage.leavitt.com/job-role-types-manager">
 				<iron-icon icon="nav:memory"></iron-icon>Job Role Types
 			</a>
-			<a name="name-changer" parent-category="2" display-role="Name Changer Admin" href$="https://[[devPrefix]]namechanger.leavitt.com">
+			<a name="name-changer" parent-category="2" display-role="Name Changer Admin"
+				href$="https://[[devPrefix]]namechanger.leavitt.com">
 				<iron-icon icon="nav:name-change"></iron-icon>Name Changer
 			</a>
-			<a name="people-search" parent-category="2" display-role="People Search User" href$="https://[[devPrefix]]peoplesearch.leavitt.com">
+			<a name="people-search" parent-category="2" display-role="People Search User"
+				href$="https://[[devPrefix]]peoplesearch.leavitt.com">
 				<iron-icon icon="nav:search"></iron-icon>People Search
 			</a>
-			<a name="employee-termination-form" parent-category="2" display-role="Terminate Employee" href$="https://[[devPrefix]]manage.leavitt.com/employee-termination-form">
+			<a name="pcms" parent-category="2" display-role="PCMS User Access" href$="https://[[devPrefix]]pcms.leavitt.com">
```

Comment:
This needs "PCMS Global Admin" as well. Not sure if it will allow you to comma separate them or how to do it